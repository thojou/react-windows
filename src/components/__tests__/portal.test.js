import React from 'react';
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Portal from "../portal";

describe('Portal', () => {
    beforeAll(() => {
        ReactDOM.createPortal = jest.fn((element, node) => {
            return element;
        })
    });

    afterEach(() => {
        ReactDOM.createPortal.mockClear();
    });

    it('should render', () => {
        const tree = renderer.create(<Portal><div /></Portal>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});