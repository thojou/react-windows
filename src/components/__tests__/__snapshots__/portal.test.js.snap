// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portal should render 1`] = `<div />`;
